T1192 : 1 line bug fix in Visualizer, and setup timer for average frame rate measurement in firmware

eppisai opened the pull request:
hi, 
this pull request is regarding code review of some small changes that i have made, so can get them merged and proceed with tasks further

<h2>changes made</h2> 
<ul>
<li>included the cpp files of widget to cmake of visualizer, so it works again</li>
<li>added timer for avg framerate measurement</li>
</ul>

eppisai committed:
Update Main.cpp

eppisai committed:
Added timer and fixed visualizer bug

eppisai committed:
minor comment improved in timer 2

BAndiT1983 reviewed (commented):
Bad formatting, empty lines are missing or too many.

eppisai committed:
formatting corrected

have added an empty line, at the start of InitTimer2 , and added a empty line before configGPIO() . Removed empty lines from int main() start and in interupt for timer2

eppisai commented:
> Bad formatting, empty lines are missing or too many.

oh..that was quick.. thank you so much, @BAndiT1983  is above okay?
